lib/ZcapClient.js

```diff
@@ -46,16 +53,19 @@ export class ZcapClient {
    *   `.sign()` function and `id` and `controller` properties that will be
    *   used for signing requests. `invocationSigner` and `delegationSigner`, or
    *   `didDocument` and `keyPairs` must be provided.
+   * @param {object} [options.suite] - The default signature suite to use when
+   *   digitally signing delegations. Default: Ed25519Signature2018.
    *
    * @returns {ZcapClient} - The new ZcapClient instance.
    */
   constructor({
     baseUrl, defaultHeaders = {}, agent, didDocument, keyPairs,
-    invocationSigner, delegationSigner
+    invocationSigner, delegationSigner, suite = Ed25519Signature2018
```

Comment:
Shouldn't this be something like `suiteClass`? This isn't an instance of a suite.